hpx/runtime/components/server/abstract_migration_support.hpp

```diff
@@ -135,5 +135,4 @@ namespace hpx { namespace components
             return this->base_type::on_migrated();
         }
     };
-}}
-
+}}    // namespace hpx::components
```

Comment:
Should we format these files at this point?

Reply:
Removed these changes.